test/integration/targets/git/tasks/checkout-new-tag.yml

```diff
@@ -48,6 +48,9 @@
     that:
       - not update_new_tag|changed
       - "'newtag' in listoftags.stdout_lines"
+  # This test fails on osx.
```

Comment:
@mattclay I won't spend time on OS X support in the near future; for the git module alone there are several things with higher priority for me.